src/cephadm/box/box.py

```diff
@@ -24,7 +27,14 @@
 # we deploy a cluster. Keep in mind that you'll be responsible of pulling the
 # image yourself with `box cluster setup`
 CEPH_IMAGE_TAR = 'docker/ceph/image/quay.ceph.image.tar'
+CEPH_ROOT = '../../../'
+DASHBOARD_PATH = '../../../src/pybind/mgr/dashboard/frontend/'
 
+root_error_msg = """
+WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING
+sudo with this script can kill your computer, try again without sudo
+if you value your time.
+"""
```

Comment:
Before this root_error_msg was made, I ran this script as root and it worked and bootstrapped a cluster. But after I killed all the boxes and tried to re-setup as a non-root user, I ran into some permissions issues (had to manually remove the  `docker/ceph/images` folder and the `loop-images` folder bc those were made previously by root).

So after I did that, I could run `./box.py -v cluster setup` as non-root user. But when I tried to run `./box.py -v cluster start`, I ran into some errors:

```
Running command: sudo chmod 777 /dev/dm-*
Added 3 logical volumes in a loopback device
Starting containers
Running command: podman-compose --podman-run-args "--group-add keep-groups --network=host --device /dev/fuse -it --device /dev/loop111 -v /dev/vg1:/dev/vg1:Z --device /dev/dm-0 --device /dev/dm-1 --device /dev/dm-2" up --scale hosts=1 -d
161a321eb307a4a51bbf152614fd49bce2da6f0b72f0145dc54f0811ebeec7a2
Running command: podman ps | \grep seed_1 | awk '{ print $1 }'

Running command: podman exec -it  hostname -i
Traceback (most recent call last):
  File "/home/melissali/Documents/ceph/src/cephadm/box/./box.py", line 358, in <module>
    main()
  File "/home/melissali/Documents/ceph/src/cephadm/box/./box.py", line 351, in main
    instance.main()
  File "/home/melissali/Documents/ceph/src/cephadm/box/util.py", line 64, in main
    function()
  File "/home/melissali/Documents/ceph/src/cephadm/box/util.py", line 70, in wrapper
    return func(*args, **kwargs)
  File "/home/melissali/Documents/ceph/src/cephadm/box/./box.py", line 245, in start
    ip = run_dc_shell_command('hostname -i', 1, 'seed')
  File "/home/melissali/Documents/ceph/src/cephadm/box/util.py", line 156, in run_dc_shell_command
    out = run_shell_command(
  File "/home/melissali/Documents/ceph/src/cephadm/box/util.py", line 119, in run_shell_command
    raise RuntimeError(f'Failed command: {command}\n{err}')
RuntimeError: Failed command: podman exec -it  hostname -i
Error: no container with name or ID "hostname" found: no such container
```
This is what the same section looks like when I run the script as root:
```
Running command: sudo chmod 777 /dev/dm-*
Added 3 logical volumes in a loopback device
Starting containers
Running command: podman-compose --podman-run-args "--group-add keep-groups --network=host --device /dev/fuse -it --device /dev/loop111 -v /dev/vg1:/dev/vg1:Z --device /dev/dm-0 --device /dev/dm-1 --device /dev/dm-2" up --scale hosts=1 -d
dc0f13e9071c3de8e654aee79d7972feaa7f07aded8583d8dd62bb8d4fc9d341
bd34af68e611b2974712029e0ed630759834dc6c7e23964d774c892c2d42a47e
3726a418aa5322daec9f098bf966b0fdf9e9deff20ae2aec6ac74f7c7ffb477b
35365fad82b9d61b55440aae4aa9ca50e7862e34ffbdca7fba322b9111316dc2
Running command: podman ps | \grep seed_1 | awk '{ print $1 }'
3726a418aa53
3726a418aa53
Running command: podman exec -it 3726a418aa53 hostname -i
10.89.0.63
```
Not really sure why it's not working as a non-root user, wondering if you know?

Reply:
I documented that issue in the docs, SElinux needs to be disabled for that to work.

Reply:
If not, you can call with `--skip-create-loop`